.cirrus.yml

```diff
@@ -22,7 +22,7 @@ task:
     # By default, Cirrus CI doesn't have HOME defined
     HOME: /root
     matrix:
-      BOX: fedora/37-cloud-base
+      BOX: bento/fedora-37
```

Comment:
Why not just use 40?

Reply:
Sure, we should document what each of these vagrant builds is trying to cover. If it is just aiming to test on a different flavor of Linux or trying to build against something older to test library compatibility.